overlord/devicestate/devicemgr.go

```diff
@@ -1789,6 +1776,8 @@ func (h fdeSetupHandler) Error(err error) error {
 
 // ResetToPostBootState is only useful for integration testing.
 func (m *DeviceManager) ResetToPostBootState() {
```

Comment:
I would move this to after Ensure, that seems the most reasonable place I think

Reply:
moved